bin/identify_kingdom.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Copyright 2023 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import json
+import logging
+import sys
+
+import requests
+from retry import retry
+
+logging.basicConfig(level=logging.INFO)
+
+
+def main(taxid, outfile):
+    kingdom = "Bacteria"
+    if not taxid.isdigit():
+        sys.exit("Invalid format of taxid {}".format(taxid))
+    try:
+        url = "https://www.ebi.ac.uk/ena/taxonomy/rest/tax-id/{}".format(taxid)
+        r = run_request(url)
+        res = json.loads(r.text)
```

Comment:
The response from this endpoint is already json, so you can just use `r.json()` to get the json thingy

Reply:
Thanks! Fixed it.